tests/integration/nm/bond_test.py

```diff
@@ -78,9 +78,27 @@ def _bond_interface(name, options):
 
 
 @nmclient_context
-def _get_bond_current_state(name):
+def _get_bond_current_state(name, option=None):
```

Comment:
I would have preferred you use plural options, but this is not a must. We can do that when the need will show itself.